core/src/test/scala/unit/kafka/security/auth/ZkAuthorizationTest.scala

```diff
@@ -197,7 +198,7 @@ class ZkAuthorizationTest extends ZooKeeperTestHarness with Logging{
       case true => "secure"
       case false => "unsecure"
     }
-    ZkSecurityMigrator.run(Array(s"--zookeeper.acl=$secureOpt", s"--zookeeper.connect=$zkConnect"))
+    ZkSecurityMigrator.run(Array(s"--zookeeper.acl=$secureOpt", s"--zookeeper.connect=$zkUrl"))
     info("Done with migration")
     for (path <- secondZk.securePersistentZkPaths) {
```

Comment:
Seems like the test hasn't really changed aside from adjusting the `zookeeper.connect` URL. It looks like we're validating the ACLs on paths in `securePersistentZkPaths`, but given this change, shouldn't we also validate that we didn't modify ACLs on other paths?